pkgs/development/python-modules/ffmpeg-normalize/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, tqdm
+, ffmpeg
+}:
+
+buildPythonPackage rec {
+  pname = "ffmpeg-normalize";
+  version = "1.15.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0161939f864e973b11d50170c657baf3e1433147f46c74a74ed5025a822e9a2d";
+  };
+
+  propagatedBuildInputs = [ ffmpeg tqdm ];
+
```

at least some guarantee that it works
```suggestion
  checkPhase = ''
    $out/bin/ffmpeg-normalize --help > /dev/null
  '';
```